docs/content/docs/install/installation.md

```diff
@@ -1,31 +1,46 @@
 ---
-title: Infrastructure install
+title: Installation
 weight: 2
 ---
-# Manual install
+# Installation
 
-To install Pipelines as Code on your cluster you simply need to run this command
-:
+## Operator Install
+
+The easiest way to install Pipelines as Code is via the [Tekton operator](https://github.com/tektoncd/operator) or with the [Red Hat Openshift Pipelines Operator](https://docs.openshift.com/container-platform/latest/cicd/pipelines/installing-pipelines.html).
```

Comment:
would be nice to add a blurb on how to install  pac via upstream tekton operator and its addons @sm43

Reply:
well pac is not yet available in tekton operator (upstream)
available only in Openshift Pipelines Operator
it is a part of tektonaddon which is enabled only for openshift platform

Reply:
ok i have updated the documentation